### PR TITLE
Add enum parsing for flox

### PR DESCRIPTION
**Pull request by nrostanml**
## Descripción 

Se agrega soporte a Flox (herramienta interna a MELI) para poder asociar un String a cada case de un enum.

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: LibraryComponents/Classes/AndesBadge/Modifier/AndesBadgeModifier.swift [hunk 1]
```diff
@@ -6,8 +6,15 @@
 import Foundation
 
 /// Used to define the type of an AndesBadge
-@objc public enum AndesBadgeModifier: Int {
+@objc public enum AndesBadgeModifier: Int, AndesEnumStringConvertible {
     case pill
 //    case dot
 //    case notification
```
[the remark this answers is not in the document]
Comment (nrostanml) [reply LibraryComponents/Classes/AndesBadge/Modifier/AndesBadgeModifier.swift:12]: no es necesario, son dos lineas que sabemos que van a estar en breves y ya tenian el naming igualado con android, x eso quedó comentado, decis que lo vuele? para mi es lo mismo lo vas a tener que agregar vos :D

File: LibraryComponents/Classes/AndesTextField/SideComponents/AndesTextFieldComponentVisibility.swift [hunk 2]
```diff
@@ -7,16 +7,14 @@
...
 
-    public static func keyFor(_ value: AndesTextFieldComponentVisibility) -> String {
+    internal static func keyFor(_ value: AndesTextFieldComponentVisibility) -> String {
         switch value {
-        case .always:
-            return "ALWAYS"
-        case .whenNotEmpty:
-            return "WHEN_NOT_EMPTY"
+        case .always: return "ALWAYS"
+        case .whenNotEmpty: return "WHEN_NOT_EMPTY"
```
[the remark this answers is not in the document]
Comment (nrostanml) [reply LibraryComponents/Classes/AndesTextField/SideComponents/AndesTextFieldComponentVisibility.swift:17]: No, good catch

File: LibraryComponents/Classes/AndesBadge/Modifier/AndesBadgeModifier.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (samuelsainz) [reply LibraryComponents/Classes/AndesBadge/Modifier/AndesBadgeModifier.swift:12]: Voto para que lo borres si jajaja pero tranqui, como vos quieras

**Pull request merged**